content/300-guides/300-upgrade-guides/200-upgrading-versions/600-upgrading-to-prisma-5.mdx

```diff
@@ -0,0 +1,677 @@
+  where: {
+    settings: {
+      path: ['someSetting'],
+      equals: someValue,
+    },
+  },
+})
+```
+
+<Admonition type="info">
+
+Note: This `path` argument change only affects PostgreSQL databases. MySQL databases are not affected as they use a different syntax.
+
+</Admonition>
+
+#### Scalar lists
+
+[Scalar list](https://www.prisma.io/docs/concepts/components/prisma-schema/data-model#scalar-fields) values must be arrays in all operations.
+
+With the following schema:
+
+```prisma
+model Post {
+  id   String   @id @default(uuid())
+  tags String[]
+}
+```
+
+Prisma 4 and lower:
+
+```js
+prisma.post.create({
+  data: {
+    tags: 'databases',
+  },
+})
+
+prisma.post.findMany({
+  where: {
+    tags: 'databases',
+  },
+})
+```
+
+Prisma 5:
+
+```js highlight=3,9;normal
+prisma.post.create({
+  data: {
+    tags: ['databases'],
+  },
+})
+
+prisma.post.findMany({
+  where: {
+    tags: ['databases'],
+  },
+})
+```
+
+#### Composite lists
+
+Operations on lists of [Composite](/concepts/components/prisma-schema/data-model#defining-composite-types) types now only accept array values.
+
+<Admonition type="info">
+
+Composite lists are only available in the [MongoDB provider](/concepts/database-connectors/mongodb)
+
+</Admonition>
+
+With the following schema:
+
+```prisma
+model Post {
+  id           String    @id @default(uuid())
+  commentsList Comment[]
+}
+
+type Comment {
+  text String
+}
+```
+
+Prisma 4 and lower:
+
+```js
+prisma.post.findMany({
+  where: {
+    commentsList: {
+      equals: { text: 'hello' },
+    },
+  },
+})
+```
+
+Prisma 5:
+
+```js highlight=4;normal
+prisma.post.findMany({
+  where: {
+    commentsList: {
+      equals: [{ text: 'hello' }],
+    },
+  },
+})
+```
+
+<details><summary>Shorthand notation usage</summary>
+
+If you use the shorthand notation and exclude `equals`, you still must supply an array value for composite list fields.
+
+Prisma 4 and lower:
+
+```js
+prisma.post.create({
+  data: {
+    commentsList: { text: 'hello' },
+})
+
+prisma.post.findMany({
+  where: {
+    commentsList: { text: 'hello' },
+  },
+})
+```
+
+Prisma 5:
+
+```js highlight=3,8;normal
+prisma.post.create({
+  data: {
+    commentsList: [{ text: 'hello' }],
+})
+
+prisma.post.findMany({
+  where: {
+    commentsList: [{ text: 'hello' }],
+  },
+})
+```
+
+</details>
+
+### <inlinecode>cockroachdb</inlinecode> provider is now required when connecting to a CockroachDB database
+
+With Prisma version 5.0.0, we are requiring the CockroachDB instances be connected to with a `cockroachdb` provider. Previously, we had accepted `postgresql` as well, but we are removing that option.
+
+If you were using [native database types](/reference/api-reference/prisma-schema-reference#postgresql) and also the `postgresql` provider, you will most likely need to `diff` your database to see if anything has changed:
+
+1. Update your provider from `postgresql` to `cockroachdb`
+2. Backup your existing `prisma.schema` file
```

Comment:
I would switch these 2 around - first to the backup, then start messign with it.